Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Build hw-event-proxy binaries
+# FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
+FROM docker.io/openshift/origin-release:golang-1.15 AS go-builder
```

Comment:
Any particular reason for not using the internal openshift registry golang builder?

Reply:
I had trouble acces openshift registry so made this temporary change. Changed back.